lib/Notes/NotePopupModal/NotePopupModal.js

```diff
@@ -161,6 +162,13 @@ const NotePopupModal = ({
 
   const hideDelete = !stripes.hasPerm('ui-notes.item.delete');
 
+  const metaSection = (
```

Comment:
I believe it's better to not show ViewMetaData when note or metadata are undefined

Reply:
@NikitaSedyx 
Implemented the change. Please review